code/modules/client/preferences.dm

```diff
@@ -350,7 +356,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 				dat += "[TextPreview(flavor_text)]...<br>"
 			dat += "<br>"
 
-			if((species in list("Unathi", "Vulpkanin", "Tajaran"))) //Species that have head accessories.
+			if(species in list("Unathi", "Vulpkanin", "Tajaran", "Machine")) //Species that have head accessories.

```

Comment:
removing this set of brackets caused no issues whatsoever, they were just extra and I removed them like I was told to do with extra brackets